conf/routes

```diff
@@ -6,6 +6,7 @@ GET	/campaign/:id		              controllers.App.index(id)
 GET	/capiImport		                controllers.App.index(id = "")
 GET	/management/analytics         controllers.App.index(id = "")
 GET	/glossary                     controllers.App.index(id = "")
+GET	/benchmarks                   controllers.App.index(id = "")
```

Comment:
Is there a reason that the routes are set up like this (with `id` set to the empty string?). Looks like the `id` should be optionally part of the request path or just parsed from the path within `controllers.App.index` handler.

Reply:
Yes. When your in a single page application and using react-router you never make a request away from the page to the server. But if you were on say `/foo` and you refreshed it would hit the server and at that point Play's router. At this point you want the server to send you back to the single page application and back into the client side router so the right page is shown to the user.